PowerPointLabs/PowerPointLabs/CropLab/CropLabSettingsDialog.xaml

```diff
@@ -0,0 +1,59 @@
+            <RadioButton x:Name="topAnchorRadioButton" Content="" HorizontalAlignment="Left" Margin="10,6,0,0"  VerticalAlignment="Top"
+                         IsChecked="{Binding Path=SelectedAnchor, Converter={StaticResource EnumToBooleanConverter}, ConverterParameter={x:Static local:AnchorPosition.Top}}" />
+            <RadioButton x:Name="topRightAnchorRadioButton" Content="" HorizontalAlignment="Left" Margin="10,6,0,0" VerticalAlignment="Top"
+                         IsChecked="{Binding Path=SelectedAnchor, Converter={StaticResource EnumToBooleanConverter}, ConverterParameter={x:Static local:AnchorPosition.TopRight}}" />
+            <RadioButton x:Name="middleLeftAnchorRadioButton" Content="" HorizontalAlignment="Left" Margin="10,6,0,0" VerticalAlignment="Top"
+                         IsChecked="{Binding Path=SelectedAnchor, Converter={StaticResource EnumToBooleanConverter}, ConverterParameter={x:Static local:AnchorPosition.MiddleLeft}}" />
+            <RadioButton x:Name="middleAnchorRadioButton" Content="" HorizontalAlignment="Left" Margin="10,6,0,0" VerticalAlignment="Top"
+                         IsChecked="{Binding Path=SelectedAnchor, Converter={StaticResource EnumToBooleanConverter}, ConverterParameter={x:Static local:AnchorPosition.Middle}}" />
+            <RadioButton x:Name="middleRightAnchorRadioButton" Content="" HorizontalAlignment="Left" Margin="10,6,0,0" VerticalAlignment="Top"
+                         IsChecked="{Binding Path=SelectedAnchor, Converter={StaticResource EnumToBooleanConverter}, ConverterParameter={x:Static local:AnchorPosition.MiddleRight}}" />
+            <RadioButton x:Name="bottomLeftAnchorRadioButton" Content="" HorizontalAlignment="Left" Margin="10,6,0,0" VerticalAlignment="Top"
+                         IsChecked="{Binding Path=SelectedAnchor, Converter={StaticResource EnumToBooleanConverter}, ConverterParameter={x:Static local:AnchorPosition.BottomLeft}}" />
+            <RadioButton x:Name="bottomAnchorRadioButton" Content="" HorizontalAlignment="Left" Margin="10,6,0,0" VerticalAlignment="Top"
+                         IsChecked="{Binding Path=SelectedAnchor, Converter={StaticResource EnumToBooleanConverter}, ConverterParameter={x:Static local:AnchorPosition.Bottom}}" />
+            <RadioButton x:Name="bottomRightAnchorRadioButton" Content="" HorizontalAlignment="Left" Margin="10,6,0,0" VerticalAlignment="Top"
+                         IsChecked="{Binding Path=SelectedAnchor, Converter={StaticResource EnumToBooleanConverter}, ConverterParameter={x:Static local:AnchorPosition.BottomRight}}" />
+        </WrapPanel>
+        <Button x:Name="okButton" Content="OK" HorizontalAlignment="Right" Margin="0,0,100,10" VerticalAlignment="Bottom" Width="80" IsDefault="True" Click="OkButton_Click"/>
+        <Button x:Name="cancelButton" Content="Cancel" HorizontalAlignment="Right" Margin="0,0,10,10" VerticalAlignment="Bottom" Width="80" IsCancel="True" Click="CancelButton_Click"/>
```

Comment:
Apply the property
IsDefault="True" on okButton, and
IsCancel="True" on cancelButton